service/src/main/java/org/cbioportal/genome_nexus/service/remote/VEPRegionDataFetcher.java

```diff
@@ -27,6 +27,9 @@
     private static final String MAIN_QUERY_PARAM = "variant";
     private static final String PLACEHOLDER = "VARIANT";
 
+    @Value("${gn_vep.timeout.seconds:0}")
```

Comment:
I see .. if the property is not specified it defaults to zero .. which means VEP will never internally timeout. That is why the property section is commented out below. Looks fine.